### ISSUE TITLE
how to start chrome to view project?

### ISSUE THREAD

**Issue by wangning08115** [cornerstonejs/cornerstone#437]
I first use node js project, and i clone code,  and nmp install,  then  but i dont not how to view in chrome , how can i do?

**Comment by dannyrb** [comment]
I don't believe this project launches the chrome browser with any particular entrypoint. The `start:dev` script will watch for changes and update the output bundle as you develop. Most of the examples consume that output, and can be opened in chrome. You may need to refresh as you make changes to use the updated code.

We could add a simple entrypoint with the WebpackHtmlPlugin to offer a better developer experience here.
